infrastructures/openshift/src/test/java/org/eclipse/che/workspace/infrastructure/openshift/environment/OpenShiftEnvironmentFactoryTest.java

```diff
@@ -291,6 +291,21 @@ public void exceptionOnRecipeLoadError() throws Exception {
     osEnvFactory.doCreate(internalRecipe, emptyMap(), emptyList());
   }
 
+  @Test(
+      expectedExceptions = ValidationException.class,
+      expectedExceptionsMessageRegExp =
+          "Environment can not contain two 'Service' objects with the same name 'db'")
```

Comment:
Same suggestion as for `KubernetesEnvironmentFactoryTest.java`. Again, not critical or important.

Reply:
Answered https://github.com/eclipse/che/pull/12842#discussion_r267200238